### PR TITLE
BBBSL-53: NGINX now forces ssl enabled websites to use https

### PR DESCRIPTION
**Pull request by davidpesce**
Creates a new nginx template and ENV variable for scalelite behind a proxy or load balancer.

### CODE REVIEW

**Comment by farhatahmad** [comment]
Hi @davidpesce 

Thanks for the PR. There's one part missing, which would be to force an https redirect if ssl is enabled. Can you include it as part of this PR?

This code should be added here: https://github.com/blindsidenetworks/scalelite/blob/master/nginx/conf.d/scalelite-ssl.template#L6
```
location / {
    return  301 https://$host$request_uri;
  }
```

**Comment by davidpesce** [comment]
@farhatahmad - I think this setting would relate to if you had configured scalelite with an actual SSL cert vs being behind a proxy. Right? The new template is scalelite-proxy.template

I made the change anyway because it is necessary with the SSL cert.

**Comment by davidpesce** [comment]
@jfederico - should I resolve the conflict?

**Comment by farhatahmad** [comment]
Hi @davidpesce,

Yes, please resolve the conflict

**Comment by davidpesce** [comment]
@farhatahmad is there a timeline for when v1.0.2 will be released? If today, I'll switch gears and resolve this.

**Comment by farhatahmad** [comment]
@davidpesce Yeah the plan is to release today

**Comment by davidpesce** [comment]
@farhatahmad - all set!

File: nginx/conf.d/scalelite-ssl.template [hunk 1]
```diff
@@ -4,8 +4,11 @@ server {
 	listen 80 default_server;
 	listen [::]:80 default_server;
 
+    location / {
+        return  301 https://$host$request_uri;
+      }
+
 	include conf.d/scalelite/*;
-	include conf.d/scalelite.common;
```
Comment (farhatahmad) [inline nginx/conf.d/scalelite-ssl.template:8]: Did you mean to remove this?

Comment (davidpesce) [reply nginx/conf.d/scalelite-ssl.template:8]: Yeah, I was following @jfederico 's lead on it. The individual file reference was removed in his latest commit.

Comment (farhatahmad) [reply nginx/conf.d/scalelite-ssl.template:8]: Ah my mistake. I got confused with something else

**Pull request merged**